forms/nht/grantsAndOthersProgressReport.json

```diff
@@ -16856,6 +16856,809 @@
+                "validate": "required"
+              },
+              {
+                "dataType": "text",
+                "name": "imageTypes",
+                "constraints": [
+                  "Ground-based photographs",
+                  "Panoramic photographs",
+                  "Low oblique aerial photographs",
+                  "Time lapse photographs",
+                  "Time lapse video",
+                  "Drone video"
+                ],
+                "validate": "required"
+              },
+              {
+                "dataType": "date",
+                "name": "imageDateCollected",
+                "validate": "required,past[now]"
+              },
+              {
+                "dataType": "time",
+                "name": "imageTimeCollected",
+                "validate": "required"
+              },
+              {
+                "dataType": "text",
+                "name": "georeferenced",
+                "validate": "required,max[100]"
+              },
+              {
+                "dataType": "text",
+                "name": "responsibleOrg",
+                "validate": "required,max[100]"
+              },
+              {
+                "dataType": "feature",
+                "name": "imageCollectedLocation",
+                "description": "Mapping field to be done and Area (ha) field be auto-calculated from mapping.",
+                "validate": "required"
+              },
+              {
+                "computed": {
+                  "expression": "$geom.areaHa(imageCollectedLocation)"
+                },
+                "dataType": "number",
+                "name": "calculatedAreaTreatedHa",
+                "units": "ha",
+                "decimalPlaces": 3
+              },
+              {
+                "computed": {
+                  "expression": "$geom.lengthKm(imageCollectedLocation)"
+                },
+                "dataType": "number",
+                "name": "calculatedLengthTreatedKm",
+                "units": "km",
+                "decimalPlaces": 3,
+                "behaviour": [
+                  {
+                    "condition": "lengthTreatedKm > 0",
+                    "type": "conditional_validation",
+                    "value": {
+                      "message": "The managed length must be mapped",
+                      "validate": "required,min[0.0001]"
+                    }
+                  }
+                ]
+              },
+              {
+                "defaultValue": {
+                  "type": "computed",
+                  "expression": "calculatedAreaTreatedHa"
+                },
+                "dataType": "number",
+                "name": "areaTreatedHa",
```

Comment:
Same with this, I think the question was area covered/display in images?  so areaCoveredHa would be more appropriate.